### ISSUE TITLE
NVDA can not read Accessibility View when screen mode not turned on

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#186401]
Testing #186218

* Turn off screen reader mode
* Go to Panel Chat
* NVDA reads "Press alt+f1 to open accessibility view"
* Press alt+f1
* NVDA reads the editor is not accessible at this time ... the editor has multiple content ... blank

IMHO this defeats some of the purpose of the accessible view, which should always be accessible (can be read by NVDA)

**Comment by meganrogge** [comment]
When using a screen reader, screen reader mode will be enabled automatically. I don't think there's a use case for someone to explicitly disable that..

**Comment by rebornix** [comment]
> When using a screen reader, screen reader mode will be enabled automatically

@meganrogge if this guarantees to work 100%, then we don't need `This editor is not accessible, press alt+f1 to open accessibility view`, and also we should not have the setting to disable it. I'm not sure how we reach this design but my hypnosis is screen reader is not always detected successfully. Asking users to view an inaccessible view when we find the editor is not accessible is a bug IMHO.

**Comment by meganrogge** [comment]
In your steps above to repro, you disabled screen reader mode. A screen reader user won't do that. That's what I was suggesting is unlikely to happen and thus we don't need to design for

**Comment by meganrogge** [comment]
If there's an issue with detection, that will be reported and can be addressed. But it should work all of the time.

**Comment by meganrogge** [comment]
@rebornix and I discussed offline and think the right move here would be if a user is in any accessibility help menu AND they’re not in screen reader optimized mode, we inform them of this and tell them an easy way to enable that from within the help menu. 

The editor’s accessibility help menu does this already. so we could simply make that a part of the generic accessibility help menu experience.

**Comment by meganrogge** [comment]
We will also now update the help text to reflect the changes - screen reader mode and accessibility hint

**Comment by meganrogge** [comment]
IMO, we should only indicate if the app isn't in screen reader optimized mode to avoid the noise. Curious to get some feedback on this from @rperez030 and @jooyoungseo.

For now, I'll go with having it everywhere.

**Comment by rperez030** [comment]
I believe that announcement should be present in places where it matters, and the place where it matters the most is the editor. The rest of VS Code is pretty accessible even when screen reader mode is off. I believe, or at least that is my hope, that a day will come when that setting will no longer be necessary. I wonder if there are customers that use screen reader for support that have that setting off and may be affected by the constant announcement. For example, customer that rely on Zoomtext Fusion.

**Comment by VSCodeTriageBot** [comment]
Issue marked as unreleased but unable to locate closing commit in issue timeline. You can manually reference a commit by commenting `\closedWith someCommitSha`, or directly add the `insiders-released` label if you know this has already been releaased
